lib/cable_ready/identifiable.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require "action_view"
```

Comment:
I guess we need to require `active_record` here and probably also define `action_view` and `active_record` as dependencies of the gem.